src/components/tabs/index.tsx

```diff
@@ -47,16 +48,29 @@ const Tabs = ({
 	const tabItems = useTabItems({ children, activeTabIndex, initialActiveIndex })
 
 	/**
-	 * useSyncedTabGroups hooks into TabProvider,
-	 * and keeps activeTabIndex & activeTabGroup in sync.
+	 * Handle syncing `activeTabIndex` and `activeTabGroup`.
 	 *
 	 * Note: only works where tabs have groups, eg <Tab group="some-string" />.
 	 */
-	const setSyncedActiveTabIndex = useSyncedTabGroups({
-		activeTabIndex,
-		setActiveTabIndex,
-		tabItems,
-	})
+	const { activeTabGroup, setActiveTabGroup } = useTabGroups()
```

Comment:
I think this is causing errors where it previously didn't, possibly because it was previously nested within another hook? 🤔 Seems like we either need to add `<TabProvider />` to a bunch of places (`/boundary/api-docs`, mostly, i just skimmed the Vercel logs)... or maybe warn rather than throw in `useTabGroups` 🤔

Reply:
I updated the `TabProvider` rendering in https://github.com/hashicorp/dev-portal/pull/1077/commits/5ad955d20e827e33f60323c8ca175664c7b1475c! It's now rendered in `CoreDevDotLayout`. I think we want to throw an error in the context hook like we do in other context hooks, and I don't think moving the rendering of the provider up in the tree should have issues. Do you forsee any? 🤔

Reply:
FWIW, I think it's valid that we support rendering `Tabs` outside of the `TabGroupsProvider`. It seems like a "progressive enhancement" sort of thing, not necessarily a hard requirement for using tabs.

That being said, there shouldn't be any practical issue with lifting the provider up, it just seems a little heavy handed that we can't use `<Tabs>` without also rendering the provider.

Reply:
Updated in https://github.com/hashicorp/dev-portal/pull/1077/commits/952c344ea4ef3a8d2e3c7438a757506919f736ad